soh/soh/Enhancements/timesaver_hook_handlers.cpp

```diff
@@ -112,6 +112,8 @@ void RateLimitedSuccessChime() {
 }
 
 bool ForcedDialogIsDisabled(ForcedDialogMode type) {
+    auto test = CVarGetInteger(CVAR_ENHANCEMENT("TimeSavers.SkipForcedDialog"),0);
+    auto test2 = test & type;
```

Comment:
These don't need to be in here anymore, do they?

Reply:
Oops, leftovers from messing with the button issue in jabu